npm/translatePolicy.go

```diff
@@ -200,11 +202,13 @@ func craftPartialIptablesCommentFromSelector(ns string, selector *metav1.LabelSe
 		}
 	}
 
-	for i, _ := range labelsWithoutOps {
-		comment += prefix + ops[i] + labelsWithoutOps[i]
-		comment += "-AND-"
+	comments := []string{}
+	for i := range labelsWithoutOps {
+		comments = append(comments, prefix+ops[i]+labelsWithoutOps[i])
 	}
 
+	sort.Strings(comments)
+	comment := strings.Join(comments, "-AND-") + "-AND-"
```

Comment:
are you appending `-AND-` but then immediately chomping it off below when returning `[:len - len("-AND-")]`?

Reply:
Correcting that.